app/Labels/LabelServiceProvider.php

```diff
@@ -100,12 +101,16 @@ private function setUpReadModels(Application $app): void
     {
         $app[self::JSON_READ_REPOSITORY] = $app->share(
             function (Application $app) {
+                $labels = file_exists(__DIR__ . '/../../excluded_labels.yml') ?
+                    file_get_contents(__DIR__ . '/../../excluded_labels.yml') : [];
```

Comment:
Note that that `YamlServiceProvider` also has a feature to [combine multiple Yaml files into one](https://github.com/cultuurnet/udb3-silex/blob/938a900ef35f4422a791f8a8ebad972982bc3ab5/app/Yaml/YamlConfigServiceProvider.php#L41-L47) it seems.

It is also used to "import" the list of god users into the config: https://github.com/cultuurnet/udb3-vagrant/blob/main/config/udb3-silex/config.yml#L2

So maybe it would even be better to use that feature to "import" the list of excluded labels into `$app['config']` and then inject it from that into `YamlExcludedLabelsRepository`? That way we don't have multiple approaches for the same problem.

Reply:
I indeed forgot the `Symfony\Component\Yaml\Yaml::parse()` method after a small refactor.

But it was intentional to avoid using the import for the labels, because if the list would get big the labels make the config also big.

Reply:
This should be fixed now.